templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -241,52 +241,58 @@
   </table>
 
   {if !$contributionMode}
-    <fieldset class="payment-details_group">
```

Comment:
This is obviously the main part. I have mixed feelings about the re-emergence of the accordian - but I don't presume to be the best opinion on that & aesthetic changes can happen later